README.md

```diff
@@ -1,12 +1,8 @@
-**BUILD:**     [![Build Status](https://dfe-ssp.visualstudio.com/Become-A-Teacher/_apis/build/status/Apply/apply-for-postgraduate-teacher-training?branchName=master)](https://dfe-ssp.visualstudio.com/Become-A-Teacher/_build/latest?definitionId=49&branchName=master)
-
-**DEPLOY:** [![Build Status](https://dfe-ssp.visualstudio.com/Become-A-Teacher/_apis/build/status/Apply/apply-for-postgraduate-teacher-training-releases?branchName=master)](https://dfe-ssp.visualstudio.com/Become-A-Teacher/_build/latest?definitionId=325&branchName=master)
-
 # Apply for teacher training
 
-A service for candidates to apply for initial teacher training.
+A service for candidates to apply for teacher training. We're currently in beta.
 
-## Environments
+## Live environments
```

Comment:
It would be useful to include the mapping of Azure environment names in this table (`d01` is `qa`, `p01` is `production`, etc)